### ISSUE TITLE
Not prompt to put Hekate as reboot to playload

### ISSUE THREAD

**Issue by hqbdv** [HamletDuFromage/aio-switch-updater#249]
**Describe the bug**

Not prompt to put Hekate as reboot to playload
**To Reproduce**
Update Atmosphere and / or Hekate through AIO Switch updater.

**System Firmware Version and CFW Version**
 - System Firmware: 16.02
 - CFW Version: Atmosphère 1.5.2

**SD card format**
exFAT

**Additional context**
Just tried to update through regular ways, it goes with the regular prompt of replacing the Ini files, but I was never asked if I want hekate as reboot to playload. tried with the Pack atmosphere + hekate and only Hekate. 
It does generate the update.bin in /bootloader so I could do a "copy_files.txt" but I have seen that the prompt should be there

**Comment by HamletDuFromage** [comment]
Yeah, you should definitely be prompted https://github.com/HamletDuFromage/aio-switch-updater/blob/master/source/extract.cpp#L129-L134

I'll have to take a look
